hummingbot/core/data_type/order_book_tracker_entry.py

```diff
@@ -26,3 +35,188 @@ def timestamp(self) -> float:
+        return self._active_order_tracker
+
+
+class BittrexOrderBookTrackerEntry(OrderBookTrackerEntry):
+    def __init__(
+        self, trading_pair: str, timestamp: float, order_book: OrderBook, active_order_tracker: BittrexActiveOrderTracker
+    ):
+        self._active_order_tracker = active_order_tracker
+        super(BittrexOrderBookTrackerEntry, self).__init__(trading_pair, timestamp, order_book)
+
+    def __repr__(self) -> str:
+        return (
+            f"BittrexOrderBookTrackerEntry(trading_pair='{self._trading_pair}', timestamp='{self._timestamp}', "
+            f"order_book='{self._order_book}')"
+        )
+
+    @property
+    def active_order_tracker(self) -> BittrexActiveOrderTracker:
+        return self._active_order_tracker
+
+
+class BitcoinComOrderBookTrackerEntry(OrderBookTrackerEntry):
+    def __init__(
+        self, trading_pair: str, timestamp: float, order_book: OrderBook, active_order_tracker: BitcoinComActiveOrderTracker
+    ):
+        self._active_order_tracker = active_order_tracker
+        super(BitcoinComOrderBookTrackerEntry, self).__init__(trading_pair, timestamp, order_book)
+
+    def __repr__(self) -> str:
+        return (
+            f"BitcoinComOrderBookTrackerEntry(trading_pair='{self._trading_pair}', timestamp='{self._timestamp}', "
+            f"order_book='{self._order_book}')"
+        )
+
+    @property
+    def active_order_tracker(self) -> BitcoinComActiveOrderTracker:
+        return self._active_order_tracker
+
+
+class LiquidOrderBookTrackerEntry(OrderBookTrackerEntry):
+    def __init__(self, trading_pair: str, timestamp: float, order_book: OrderBook):
+        self._trading_pair = trading_pair
+        self._timestamp = timestamp
+        self._order_book = order_book
+
+    def __repr__(self) -> (str):
+        return (
+            f"LiquidOrderBookTrackerEntry(trading_pair='{self._trading_pair}', timestamp='{self._timestamp}', "
+            f"order_book='{self._order_book}')"
+        )
+
+    @property
+    def trading_pair(self) -> (str):
+        return self._trading_pair
+
+    @property
+    def timestamp(self) -> (float):
+        return self._timestamp
+
+    @property
+    def order_book(self) -> (OrderBook):
+        return self._order_book
+
+
+class HitBTCOrderBookTrackerEntry(OrderBookTrackerEntry):
+    def __init__(
+        self, symbol: str, timestamp: float, order_book: OrderBook, active_order_tracker: HitBTCActiveOrderTracker
+    ):
+        self._active_order_tracker = active_order_tracker
+        super(HitBTCOrderBookTrackerEntry, self).__init__(symbol, timestamp, order_book)
+
+    def __repr__(self) -> str:
+        return (
+            f"HitBTCOrderBookTrackerEntry(symbol='{self._symbol}', timestamp='{self._timestamp}', "
+            f"order_book='{self._order_book}')"
+        )
+
+    @property
+    def active_order_tracker(self) -> HitBTCActiveOrderTracker:
+        return self._active_order_tracker
```

Comment:
Similar to `OrderBookMessage`, each exchange will now have their own `OrderBookTrackerEntry` in their respective folders.